Autofac.Extras.Moq.sln

```diff
@@ -1,8 +1,17 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26730.16
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{E3CCAB0F-900D-48DF-A42C-963E56973E4E}"
+	ProjectSection(SolutionItems) = preProject
+		appveyor.yml = appveyor.yml
+		build.ps1 = build.ps1
+		CONTRIBUTING.md = CONTRIBUTING.md
```

Comment:
Let's delete CONTRIBUTING.md and ISSUE_TEMPLATE.md from the repo so it'll fall back to the common .github repo.